tsl/test/expected/dist_hypertable-14.out

```diff
@@ -3796,15 +3795,15 @@ ANALYZE hyper_estimate;
 EXPLAIN (COSTS ON)
 SELECT *
 FROM hyper_estimate;
-                                      QUERY PLAN                                      
---------------------------------------------------------------------------------------
- Append  (cost=100.00..607.58 rows=15 width=20)
-   ->  Foreign Scan on _dist_hyper_16_38_chunk  (cost=100.00..101.40 rows=4 width=20)
-   ->  Foreign Scan on _dist_hyper_16_39_chunk  (cost=100.00..101.20 rows=2 width=20)
-   ->  Foreign Scan on _dist_hyper_16_40_chunk  (cost=100.00..101.30 rows=3 width=20)
-   ->  Foreign Scan on _dist_hyper_16_41_chunk  (cost=100.00..101.20 rows=2 width=20)
-   ->  Foreign Scan on _dist_hyper_16_42_chunk  (cost=100.00..101.30 rows=3 width=20)
-   ->  Foreign Scan on _dist_hyper_16_43_chunk  (cost=100.00..101.10 rows=1 width=20)
+                                        QUERY PLAN                                        
+------------------------------------------------------------------------------------------
+ Append  (cost=10000.00..60021.38 rows=15 width=20)
```

Comment:
Why have the cost ranges shifted from 100 to 10000 ranges?

Reply:
That's because I significantly increased the default fdw tuple cost. It was unreasonably low, lower than even the parallel tuple cost. I think now the plans are better, we started to prefer aggregation pushdown in more cases.